.pre-commit-config.yaml

```diff
@@ -4,28 +4,45 @@
 # To run: `pre-commit run --all-files`
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.4.0
     hooks:
       - id: check-added-large-files
       - id: debug-statements
       - id: mixed-line-ending
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 22.10.0
     hooks:
       - id: black
         language_version: python3
         exclude: versioneer.py
         args: [--target-version=py38]
         files: ^python/
   - repo: https://github.com/PyCQA/flake8
-    rev: 3.8.4
+    rev: 6.0.0
     hooks:
       - id: flake8
-        args: [--config=python/.flake8]
-        files: ^python/
+        args: ["--config=setup.cfg"]
+        files: python/.*$
+        types: [file]
+        types_or: [python] # TODO: Enable [python, cython]
```

Comment:
I will enable flake8 on Cython files in a follow-up PR. Commit 2e4f34fa354aff160b87f4e011a5b5c3349cb55c is what I'll put in that PR.